backend/src/main/java/com/exadel/carinsurance/mapper/AssignmentMapper.java

```diff
@@ -0,0 +1,56 @@
+package com.exadel.carinsurance.mapper;
+
+import com.exadel.carinsurance.model.assignment.AssignmentEntity;
+import com.exadel.carinsurance.model.assignment.ContactInfoEntity;
+import com.exadel.carinsurance.model.request.AssignmentRequestEntity;
+import com.exadel.carinsurance.model.response.AssignmentResponseEntity;
+import com.exadel.carinsurance.model.response.ContactInfoResponseEntity;
+import com.exadel.carinsurance.model.response.DirectionOfImpactResponseEntity;
+import com.exadel.carinsurance.model.response.VehicleConditionInfoResponseEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AssignmentMapper {

```

Comment:
It is good to have a single interface for as much mappers as possible. Please create a common generic interface for your mappers.

Reply:
Created it. Done